src/include/network/itp/itp_protocol_interpreter.h

```diff
@@ -85,7 +85,12 @@ class ITPProtocolInterpreter : public ProtocolInterpreter {
   void SetPacketMessageType(common::ManagedPointer<ReadBuffer> in) override;
 
  private:
+  InputPacket curr_input_packet_{};
+  std::unordered_map<std::string, std::string> cmdline_options_;
```

Comment:
Please add a comment describing this + I haven't seen how this is used yet, but I'm suspicious of keeping cmdline options here. It should probably be a setting.